### ISSUE TITLE
Blank links for screenreader on past issues

### ISSUE THREAD

**Issue by benaltair** [bahaistudies/jbs#57]
![image](https://user-images.githubusercontent.com/17771229/116486166-cec09400-a841-11eb-8478-248b1ba74a28.png)

Reads "Blank" for each cover. Need to add `aria-hidden` to the covers I think. Also wrap them in `role=list` or similar, to help with navigating the large feed of items.

**Comment by benaltair** [comment]
Odd, `aria-hidden` is already present.
```html
<a href="https://journal.bahaistudies.ca/online/issue/view/8" title="Vol. 27 No. 3 (2017)" aria-hidden="true">
	<img class="card-img-top issue-summary-cover" src="https://journal.bahaistudies.ca/public/journals/1/cover_issue_9_en_US.jpg" alt="The Colors at Sunset - artwork by Roger Bansemer">
</a>
```

**Comment by benaltair** [comment]
So `aria-hidden` doesn't work on focusable elements. 

> You must never use `aria-hidden="true"` on any focusable element, or on any element that itself contains focusable children.
https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/hidden/#does-not-work-on-focusable-elements

Instead I added `aria-label` to the cover image and moved `aria-hidden="true"` to the text link. Then to make sure the text wasn't tab-selectable I added `tabindex="-1"` to the element. Not sure at this point if it should be inverse, with the image unselected and the text selected. I think this is the most intuitive for now.
